var/spack/repos/builtin/packages/xerces-c/package.py

```diff
@@ -101,4 +101,7 @@ def configure_args(self):
         if 'transcoder=windows' in spec:
             args.append('--enable-transcoder-windows')
 
+        if 'transcoder=none' in spec:
+            args.append('--without-icu')
```

Comment:
This should be an `elif` since multi-values aren't allowed... can you change the others as well? This whole block would also be much more robust against typos as:
```python
transcoder = spec.variants['transcoder'].value
if transcoder == 'none':
    args.append('--without-icu')
elif transcoder == 'icu':
    args.extend(['--enable-transcoder-icu', '--with-icu=' + spec['icu4c'].prefix])
elif transcoder == 'macos':
    args.append('--enable-transcoder-macosunicodeconverter')
else:
    args.append('--enable-transcoder-' + transcoder)
```